src/test/files/checks/htl/HtlCommentsCheck.html

```diff
@@ -1,5 +1,17 @@
-<!--/* Good, this comment will not be rendered */-->
```

Comment:
What about supporting Apache SSI?
http://httpd.apache.org/docs/current/mod/mod_include.html

Reply:
Please take a low now: https://github.com/Cognifide/AEM-Rules-for-SonarQube/pull/166/commits/932e208985e313b589307d9eff36f01e2771220c